test/core/test_float_builtins.c

```diff
@@ -34,6 +34,7 @@ int test_builtins() {
   TEST(fmin)
   TEST(fmod)
   TEST(scalbn)
+  TEST(ldexp)
```

Comment:
Why add ldexp in this PR specifically?

Reply:
Because that is reproducer of this the failure.  See https://github.com/emscripten-core/emscripten/issues/19781#issuecomment-1711720919